src/git.ts

```diff
@@ -264,8 +264,9 @@ export function getCurrentHash(cwd: string) {
 /**
  * Get the commit hash in which the file was first added.
  */
-export function getFileAddedHash(filename: string, cwd: string) {
-  const results = git(["rev-list", "HEAD", filename], { cwd });
+export function getFileAddedHash(filename: string, cwd: string, fromRef?: string, toRef?: string) {
```

Comment:
I think it might be better to combine these in a single parameter, like `range?: { fromRef: string; toRef: string }`. That way it's clear that both parameters are required.

Reply:
Also, if you merge/rebase this, the function moved to `src/git/gitUtilities.ts`.